packages/main/test/specs/StepInput.spec.js

```diff
@@ -456,6 +456,16 @@ describe("'change' event firing", () => {
 		assert.strictEqual(Number(await changeResult.getProperty("value")), 2, "'change' event is fired 2 times");
 	});
 
+	it("Value state is not changed, when value-state-change is prevented", async () => {
+		const input = await browser.$("#stepInputValueStateChange").shadow$("ui5-input").shadow$("input");
+
+		const valueState = await input.getProperty("valueState");
+		await input.click();
```

Comment:
this test doesn't test the functionality. Basically validation is done when the focus leaves the input and here the focus remains in the input even after change.

I would suggest you to check also whether the event was fire, not only if it was prevented.